prophetess/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: "v1"
+kind: "ConfigMap"
```

Comment:
Generally, unless the value has (or in the case of template could have) a `:`, `{}`, or `[]` you'd want to quote. Otherwise I think quotes can be ignored. I'm sure there's probably a few other illegal characters.